public/locales/en-US/activate.ftl

```diff
@@ -447,12 +447,13 @@ webext-activity-resources-5 = <rLink>Extension Examples and Walkthroughs by Visw
 webext-activity-resources-6 = <rLink>Let’s Write a Web Extension by Blake Winton</rLink>
 webext-activity-resources-7 = <rLink>Create a Simple WebExtension by David Wals</rLink>
 webext-activity-resources-8 = <rLink>Browser Compatibility Table for APIs</rLink>
+webext-activity-resources-9 = <rLink>Webextensions.tech- a scaffolding tool for browser extensions</rLink> by Shivam Singhal, Tushar Arora &amp Trishul
 webext-activity-followup = Event Follow-Up
 webext-activity-followup-1 = Immediately after every event, share the link to the impact form with your attendees: <feedbackLink>mzl.la/howwasit</feedbackLink>. While you’re at it, why not also send a copy of your slide deck and invite the attendees to join the Telegram group <telegramLink>@addonchats</telegramLink>?
 webext-activity-followup-2 = Fill out the post-event metrics on the event page you set up on the Reps portal.
 webext-activity-followup-3 = <discourseLink>Tell us about your event</discourseLink> on Discourse. We love to hear about upcoming plans, and of course love to hear about what was achieved at the event with a picture or two!
 webext-activity-ack = Acknowledgements
-webext-activity-ack-1 = Thank you to Viswaprasanth Ks, Santosh Viswanatham, Trishul Goel, Karthic Keyan, Andre Garzia, Daniele Scasciafratte, and Michael Kohler.
+webext-activity-ack-1 = Thank you to Viswaprasanth Ks, Santosh Viswanatham, Trishul Goel, Karthic Keyan, Andre Garzia, Daniele Scasciafratte, Michael Kohler, Shivam Singhal and Tushar Arora.
```

Comment:
can you change the key so it triggers retranslations for the translations that are already done? More info here: https://github.com/mozilla/activate.mozilla.community#how-to-add-or-update-content

Reply:
Hi @MichaelKohler thanks for the link , made the changes :star: